meson.build

```diff
@@ -0,0 +1,86 @@
+project(
+	'dunst',
+	'c',
+	version: '1.9.2',
+	license: 'MIT',
+	meson_version: '>=0.60.0',
+	default_options: [
+		'c_std=gnu99',
+		'warning_level=1',
+		'b_ndebug=if-release',
+	],
+)
+
+add_project_arguments([
+	'-pedantic',
+	'-Wno-gnu-zero-variadic-macro-arguments',
+	'-Wno-overlength-strings',
+], language: 'c')
+
+if get_option('buildtype').startswith('debug')
+	add_project_arguments('-DDEBUG_BUILD', language: 'c')
+endif
+
+cc = meson.get_compiler('c')
+
+cairo = dependency('cairo')
+glib = dependency('glib-2.0')
+gio = dependency('gio-2.0')
+gdk_pixbuf = dependency('gdk-pixbuf-2.0')
+pangocairo = dependency('pangocairo')
+x11 = dependency('x11')
+xinerama = dependency('xinerama')
+xext = dependency('xext')
+xrandr = dependency('xrandr', version: '>=1.5')
+xscrnsaver = dependency('xscrnsaver')
+systemd = dependency('systemd', required: get_option('systemd'))
+libnotify = dependency('libnotify', required: get_option('dunstify'))
+realtime = cc.find_library('rt')
+math = cc.find_library('m')
+wayland_client = dependency('wayland-client', required: get_option('wayland'))
+wayland_protos = dependency('wayland-protocols', version: '>=1.32', required: get_option('wayland'))
```

Comment:
You want 1.12 here to fix the CI build.
This project doesn't require anything newer than 1.12, and 1.32 is not available in the ubuntu-lts CI runner.

Reply:
Yeah, just keep it on the oldest version that still works.